build.gradle.kts

```diff
@@ -79,3 +82,20 @@ dependencies {
 
     compile("com.google.protobuf:protobuf-java:2.5.0")
 }
+
+val intellijRepoChannel: String by project
+val publishPlugin: PublishTask by tasks
+publishPlugin {
```

Comment:
So as I understand, IJ release process uses `net.researchgate.release`?

Reply:
right. that is the gradle release plugin, called by `gradlew release`. This does some tagging, updates the version in `settings.gradle` to the next version, pushes a PR to github. 

To do the actual release, we need to run the Kokoro release job (not yet setup for this repo). This release job will invoke the release.sh script in the PR. (previously when we were using Jenkins we had the ability to configure Kokoro to detect a "release" triggered from the gradle release plugin and automatically call the release job. But it wasn't possible with Kokoro last I checked).